src/libs/recommendation/useHomeRecommendedIdsMutation.ts

```diff
@@ -19,14 +19,21 @@ export const useHomeRecommendedIdsMutation = () => {
           body: JSON.stringify(requestBodyParams),
         })
         if (!response.ok) {
-          throw new Error('Failed to fetch recommendation')
+          throw new Error('Failed to fetch recommendation (response not ok)')
         }
         const responseBody: RecommendedIdsResponse = await response.json()
+        if (!responseBody.playlist_recommended_offers.length) {
```

Comment:
nittpick(non-blocking): je trouve ça plus lisible et explicite d'écrire une (in)égalité plutot que de juste mettre un `!` qui est un seul petit caractère qui peut ne pas etre vu lors d'une lecture rapide et etre source de confusions

```suggestion
        if (responseBody.playlist_recommended_offers.length === 0) {
```